mcserver/admin.py

```diff
@@ -102,16 +103,23 @@ class TrialAdmin(admin.ModelAdmin):
         'status',
         'created_at', 'updated_at',
         'server', 'git_commit',
-        'processed_duration', 'processed_count',
+        'formatted_duration', 'processed_count',
         'is_meta_null',
         'trashed', 'trashed_at',
     )
     raw_id_fields = ('session',)
     inlines = [ResultInline]
+    formatted_duration.short_description = 'duration'
```

Comment:
I tried to run it locally and it is complaining about `formated_duration `and `processed_duration` not being defined:

```
  File "C:\Workspaces\opencap\opencap-api\mcserver\admin.py", line 97, in <module>
    class TrialAdmin(admin.ModelAdmin):
  File "C:\Workspaces\opencap\opencap-api\mcserver\admin.py", line 112, in TrialAdmin
    formatted_duration.short_description = 'duration'
NameError: name 'formatted_duration' is not defined
```